client/src/pages/automation/workflow-executions/components/dialog-content/WorkflowTriggerAccordion.tsx

```diff
@@ -0,0 +1,115 @@
+import {TriggerExecutionModel} from '@/middleware/helios/execution';
+import {
+    AccordionContent,
+    AccordionItem,
+    AccordionTrigger,
+} from '@radix-ui/react-accordion';
+import {CheckCircledIcon} from '@radix-ui/react-icons';
+import InlineSVG from 'react-inlinesvg';
+import ReactJson from 'react-json-view';
+import {twMerge} from 'tailwind-merge';
+
+const WorkflowTriggerAccordion = ({
+    triggerExecution,
+    triggerExecutionCompleted,
+}: {
+    triggerExecution: TriggerExecutionModel;
+    triggerExecutionCompleted: boolean;
+}) => {
+    const {endDate, id, input, output, startDate, workflowTrigger} =
+        triggerExecution;
+
+    return (
+        <>
+            {workflowTrigger?.label && (
+                <AccordionItem key={id} value={id || ''}>
+                    <AccordionTrigger className="flex w-full items-center justify-between border-gray-100 bg-white px-2 py-3 data-[state=closed]:border-b">
+                        <div className="flex items-center text-sm">
+                            {triggerExecution?.componentDefinition?.icon && (
+                                <InlineSVG
+                                    className="mr-1 h-6 w-6"
+                                    src={
+                                        triggerExecution?.componentDefinition
+                                            ?.icon
+                                    }
+                                />
+                            )}
+
+                            {workflowTrigger?.name || workflowTrigger?.type}
+                        </div>
+
+                        <div className="flex items-center">
+                            {startDate && endDate && (
+                                <span className="ml-auto mr-2 text-xs">
+                                    {Math.round(
+                                        endDate?.getTime() - startDate.getTime()
+                                    )}
```

Comment:
Isto bi spremija u varijablu. Ne sviđa mi se inline-anje metoda, mislin da je praksa koja će stvoriti konfuziju kasnije. Clean code i sve to šta bi onaj Java starac reka